miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppView.kt

```diff
@@ -0,0 +1,70 @@
+package com.rakuten.tech.mobile.miniapp.view
+
+import android.content.Context
+import android.view.View
+import com.rakuten.tech.mobile.miniapp.MiniAppInfo
+import com.rakuten.tech.mobile.miniapp.sdkExceptionForInvalidArguments
+
+/**
+ * This class can be use to create miniapp instances.
+ * @param context of where the miniapp is loaded.
+ * @param config provide the necessary configuration.
+ */
+class MiniAppView(val context: Context, val config: MiniAppConfig) {
```

Comment:
@gopi-karmakar this is a public class, we can not make it available the implementation to public. Hope you understand my concern.

Reply:
@munir-rakuten Ahh now I've understood why you have moved the implementation to the `MiniAppViewHandler.kt` class. Actually that shouldn't be the case the Implementations not concerned to the Host Apps should be hidden with the either `internal` or `private` as per the requirement. Just Like the `MiniAppView` holding the `private MiniAppViewHandler` which won't be available to the Host Apps and it's exactly the same thing what I am talking about.

`MiniAppViewHandler.kt` should only hold all the instances and no implementation so that we can pass only the Handler Object to the classes or to the functions instead a bunch of params.

Reply:
We can keep the implementation from here to there based on the responsibility of that particular class/object e.g. 
1. Responsibility of `MiniAppView` : To represent the public functions to HostApp, so adding any private implementation in this class can be optional.
2. Responsibility of `MiniAppViewHandler` : To hold all the private/internal implementations to handle the `MiniAppView`. 
It's all about separation of concern - my concern here to write a class according to the responsibility, so the code is being readable.